### DISCUSSION TITLE
✅ Add scroll offset for Anchor Widget

### DISCUSSION THREAD

**Discussion by Henzxon** [elementor/elementor#24183] [Feature Request]
### Prerequisites

- [X] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [X] The feature is still missing in the latest stable version of Elementor ( Elementor Pro )

### What problem is your feature request going to solve? Please describe.

I'm always frustrated when the anchor link widget snaps behind the header navigation this issue is also apparent for the table of contents widget or any anchor link behavior.

### Describe the solution you'd like

It would be nice if an "anchor link offset" is available for each "breakpoint" to allow customizability over the section snapping.

### Describe alternatives you've considered

It can be solved using a custom Javascript to alter the scrolling behavior

**Comment by nicholaszein** [comment]
Hello there!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor Core and Pro v3.25.0! 🥳 

Now Elementor uses Native CSS Smooth Scroll, and anchor links to prevent conflicts with other features. We also added an option to set the Anchor Offset directly in the Sticky options of the sticky element. For more information see the documentation:
https://elementor.com/help/sticky-headers/

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
